documentation/modules/ROOT/pages/configuration/outbox-event-router.adoc

```diff
@@ -184,14 +183,14 @@ After observing all those pieces we can see what the default configuration does:
 |The `id` shows up as a header in the Kafka message, this is the unique ID of the event, can be used for consumer side dedupe for instance.
 
 |`aggregatetype`
-|Is the default field for the routing, it gets append to the topic name (check configuration `route.topic.replacement`)
+|Is the default field for the routing, it gets append to the topic name (check configuration <<configuration-property-route-topic-replacement, `route.topic.replacement`>>)
 
 |`aggregateid`
 |Becomes the Kafka message key, which is important for keeping ordering within Kafka partitions
 
 |`type`
-|The `type` column becomes either part of the message envelope as `eventType` or message header. +
-See option `table.fields.additional.placement` for more details.
+|The `type` column is not part of the message envelope or headers by default.
```

Comment:
See above, this entry should be removed altogether. But it should be presented as an example for using the generic "additional fields" option.